twitter/Models/User.h

```diff
@@ -0,0 +1,20 @@
+//
+//  User.h
+//  twitter
+//
+//  Created by Nicholas Wayoe on 6/29/20.
+//  Copyright © 2020 Emerson Malca. All rights reserved.
+//
+
+#import <Foundation/Foundation.h>
+
+NS_ASSUME_NONNULL_BEGIN
+
+@interface User : NSObject
+@property (nonatomic,strong) NSString *name;
+@property (nonatomic,strong) NSString *ScreenName;
```

Comment:
Should be lowercase - `screenName`. Also, remind me to go over difference between twitter's model and your model. They do not need to match. `handle` might be more accurate here.